Update styles.css

alex-ns-nl opened the pull request:
- organised the classes in alphabetical order
- focused on DRY principle (= no duplicate code)
- added variable colours
- introduced float classes

alex-ns-nl committed:
Update styles.css

- organised the classes in alphabetical order
- focused on DRY principle (= no duplicate code)
- added variable colours
- introduced float classes

alex-ns-nl committed:
Update index.html

- updated background image in top container
- changed the font
- added 2 divs in the middleContainer

alex-ns-nl committed:
Merge pull request #4 from Zandag3007/Zandag3007-patch-2

Update index.html

anouq commented on CSS/styles.css:
```diff
@@ -1,71 +1,79 @@
-  width: 100%;
-  height: 100%;*/
+
+.positionRelative{
```
er mist een spatie tussen `positionRelative` en `{`. dit is typisch iets wat een (automatische) code formatter voor je zou oplossen. dus of door middel van een key shortcut, of een linting tool. zie ook mijn recente berichten in Slack

alex-ns-nl replied:
done! zag ook gelijk dat ik het ook bij `subDiv` ook was vergeten

anouq commented on CSS/styles.css:
```diff
@@ -1,71 +1,79 @@
+.container-h2 {
+  font-size: 1.5rem;
+}
+
```
ik zie (in de html file) dat je `container-p` toepast op alle `<p>` tags. dus dan kun je de class net zo goed niet toepassen, en in de css er het volgende van maken:

```
p {
  font-size: 1rem;
  top: 210px;
  right: 30px;
}
```

hetzelfde geldt voor `h1-container`, etc

alex-ns-nl replied:
Ja, goeie inderdaad. Volgens mij was mijn beredenering dat ik in de toekomst eventueel verschillende styles op <p> wil toepassen. Als ik er zo over nadenk kan ik dan tzt daar een nieuwe class voor aanmaken. Thanks!

anouq replied:
je hoeft eigenlijk bijna nooit in je code rekening te houden met 'wat als ik dit in de toekomst wil'. als je dat in de toekomst wil, bouw je het in de toekomst.

anders kun je eindeloos uitzonderingen en logica toevoegen, voor dingen die je potentieel in de toekomst wil. en dat zou de boel nogal onoverzichtelijk maken.

anouq commented on CSS/styles.css:
```diff
@@ -1,71 +1,79 @@
-.topContainer{
-  background-color: var(--almost-black);
-  color: white;
+
```
een voorstel/tip:

```
.mainDiv > div {
  width: 50%;
}
```

en dan heb je de class `subDiv` niet meer nodig

zie ook:

https://www.w3schools.com/cssref/sel_element_gt.asp

en er zijn meer opties. goed om deze te kennen en toe te passen:

https://www.w3schools.com/cssref/css_selectors.asp

alex-ns-nl replied:
Ah dit is cool! Precies wat ik nodig heb inderdaad :D

anouq commented:
opmerkingen over de branch naam (`Zandag3007-patch-1`):

- het is niet nodig om de repo naam in de branch naam te zetten. is dubbelop :) 
- `patch-1` is een nietszeggende naam. ik snap dat het een patch is. in dit geval zou je kunnen doen `change-styling-and-template-structure`. zo kan ik meteen zien wat hier ongeveer de bedoeling is. als je een story uit Jira oppakt weet je dat vaak vooraf natuurlijk al.

anouq commented on CSS/styles.css:
```diff
@@ -1,71 +1,79 @@
+  top: 210px;
+  right: 30px;
+}
+
```
los van of de classes `container-h1`, `container-h2` etc nodig zijn: de benamingen kloppen niet helemaal.

je zet de class `container-h1` namelijk op een `<h1>`. maar container impliceert dat *binnen* de container een h1 valt. bijvoorbeeld:

```
<div class="h1-container">
   <h1>blabla</h1>
</div>
```

alex-ns-nl replied:
Goed dat je het zegt! Ik zal de adviesen/tips in deze comment als "resolved" noteren. Als het goed is, zijn de updates dan in een nieuwe commit verwerkt

anouq commented on index.html:
```diff
@@ -1,30 +1,38 @@
-    <div class="blue">
+    <div class="skillSet subDiv">
+      <h2 class="container-h2">My Skills</h2>
+      <p class="container-p">Lorem ipsum dolor sit amet, vitae volutpat,<br>dui conubia, dolor urna.<br>Non auctor, montes nulla distinctio.</p>
```
net als met `<img />` heb je hier een tag (`<br>`) zonder closing tag. dus daar kun je dit van maken: `<br />`

alex-ns-nl replied:
Done :)

alex-ns-nl committed:
Reviewed updates

- removed/adjusted unnecessary classes: .container-p, .container-h1, .container-h2, .subDiv
- updated self-closing tags
- re-ordered classes

alex-ns-nl committed:
updated reviews index.html

- removed unnecessary classes
- updated self-closing tags

anouq reviewed (approved):
toppie!
